tests/desktop/developer_hub/conftest.py

```diff
@@ -6,7 +6,15 @@
 
 
 @pytest.fixture
-def login(mozwebqa, existing_user):
+def login_existing(mozwebqa, existing_user):
+    from pages.desktop.developer_hub.home import Home
+    home_page = Home(mozwebqa)
+    home_page.go_to_developers_homepage()
+    home_page.login(mozwebqa, existing_user['email'], existing_user['password'])
+
+
+@pytest.fixture
+def login_new(mozwebqa, existing_user):
```

Comment:
Nothing, it seems. I guess maybe a commit never made it to my Github repo or something. When I was working on it `login_new` used a `new_user` fixture, which also seems to be absent, so that tests that use `login_new` use a new user created via FxAPom. I'll look into this and fix it.